### ISSUE TITLE
Npm + browserify (can't find variable jQuery)

### ISSUE THREAD

**Issue by pygeek** [terrymun/Fluidbox#135]
It seems like your plungin isn't browserify compatible. I'm installing using the latest version of npm and browserify, but it seems like your plugin is depending on jQuery being included separately in the head.

**Comment by terrymun** [comment]
I have no experience working with Browserify. Could you recommend how this can be rectified?

**Comment by gwendoux** [comment]
I had the same issue here.
You could use [browserify-shim](https://github.com/thlorenz/browserify-shim) meanwhile. Works fine.

**Comment by mmintel** [comment]
@gwendoux i was trying it with browserify-shim but I am still getting the jQuery missing error. Can you tell me how you solved it? I am not so familiar with browserify-shim, here is my code of package.json

```JSON
  "browserify-shim": {
    "fluidbox": {
      "depends": [
        "jquery:jQuery"
      ]
    }
  }
```

**Comment by gwendoux** [comment]
@mmintel Do you include jquery in your package.json ? When you open the result of your browserify operation,  jquery is it included ?

**Comment by mmintel** [comment]
@gwendoux yes jquery is included in my dependencies, it also works with browserify.
